README.md

```diff
@@ -1 +1,91 @@
+
+- I messaggi possono essere cancellati
+- Account momentaneamente disattivato
+- Account cancellato e bannato
+
+Ogni utente ha il dovere di segnalare ogni violazione delle regole di condotta sia all'interno della community con un messaggio privato agli admin sia inviando un'email a [swiftdevelopersitalia@gmail.com](mailto:swiftdevelopersitalia@gmail.com)
+
+
+### Best Practices
+
+Per facilitare la consultazione, il confronto e la crescita della community abbiamo creato delle semplici linee guida da seguire:
+
+#### Generali
+- La lingua ufficiale della community è l'Italiano.
+- Cambia l'avatar e non lasciare quello di default e usa un nome non troppo generico. In questo modo sarai più riconoscibile all'interno della community.  Preferisci una foto reale piuttosto che un immagine fake o generata da AI.
+
+#### Prima di postare
+
+- Prima di fare una domanda e/o provare ad avviare una discussione accertati sempre di scriverla nel canale corretto. Leggi la descrizione del canale per essere sicuro che è attinente con la domanda/discussione che stai per fare.
+- Se la tua domanda non è attinente a nessuno dei canali, puoi postare in *#misc-general*.
+- Cerca di stare _in topic_. In questa community trattiamo tutto ciò che è relativo allo sviluppo alla piattaforma iOS, macOS e linguaggio Swift. Parliamo di Swift, Architetture, Librerie, Design, Conferenze, eventi, bugs e tutto ciò che è importante per uno sviluppatore dell'ecosistema Apple e Swift.
+- Questa community stimola un approccio olistico allo sviluppo mobile e accoglie discussioni che includono anche lo sviluppo Android, Kotlin, Linux, Embedded. Essere un mobile developer significa saper interagire con team cross-funzionali in cui si da valore all'unione tra devs a prescindere dalla piattaforma.
+- Se vuoi postare qualcosa che non è attinente con il tema principale della community scrivi nel canale #off-topic.
+- Se hai una domanda tecnica, cerca prima di trovare una soluzione facendo una ricerca. Se non hai trovato la soluzione, saremo felici di aiutarti (siamo qui per questo!).
+
+### Lavoro e annunci
+
+Per pubblicare una proposta lavorativa è obbligatorio usare il canale #lavoro rispettando alcune semplici indicazioni:
+
+- Nome Azienda o Tipo di azienda
+- Tipo di contratto
+- Inquadramento
+- RAL (quando è variabile descrivere il range sensato)
+- Luogo: città, regione dove il candidato svolgerà il suo lavoro
+- Remoto: specificare se il lavoro può essere svolto da remoto o una percentuale da remoto e l'altra dal datore di lavoro
+- Esperienza: specificare a parole o con titoli l'esperienza del candidato all'annuncio. Per es: junior, middle, senior o con tot anni di esperienza 
+- Tecnologia: specificare le tecnologie o il minimo di tecnologia che il candidato dovrà conoscere per partecipare alla selezione
+- Benefit: _(Opzionale)_ specificare quale tipo di benefit vengono offerti come parte del contratto di lavoro.
+              
```

Comment:
In ADI mi manca, ma lo aggiungo a breve la regola , che non si possono inserire annunci identici nel breve tempo. e' successo che hanno inserito lo stesso annuncio piu' volte. Metterei almeno 3 mesi oppure se si vuole inserire una nuova posizione occorre cambiare la RAL e/o esperienza, annuncio ricerca junior ma puoi inserire l' annuncio di middle.. e dopo 3 mesi puoi inserire lo stesso annuncio. Ho detto 3 mesi come tempo accettabile ma pensaci anche tu